src/include/optimizer/stats/column_stats.h

```diff
@@ -0,0 +1,99 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// column_stats.h
+//
+// Identification: src/include/optimizer/stats/column_stats.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sstream>
+
+#include "common/macros.h"
+#include "type/types.h"
+
+namespace peloton {
+namespace optimizer {
+
+//===--------------------------------------------------------------------===//
+// ColumnStats
+//===--------------------------------------------------------------------===//
+class ColumnStats {
+ public:
+  ColumnStats(oid_t database_id, oid_t table_id, oid_t column_id,
+              const std::string column_name, bool has_index, size_t num_rows,
+              double cardinality, double frac_null,
+              std::vector<double> most_common_vals,
+              std::vector<double> most_common_freqs,
+              std::vector<double> histogram_bounds)
+      : database_id(database_id),
+        table_id(table_id),
+        column_id(column_id),
+        column_name(column_name),
+        has_index(has_index),
+        num_rows(num_rows),
+        cardinality(cardinality),
+        frac_null(frac_null),
+        most_common_vals(most_common_vals),
+        most_common_freqs(most_common_freqs),
+        histogram_bounds(histogram_bounds),
+        is_basetable{true} {}

```

Comment:
Actually, the `ColumnStats` is just a wrapper of all column stats and we just want to use it like a `struct`. We wan to make these variables public so that we can directly access the variables instead of calling functions.